dali/test/python/test_operator_affine_transforms.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Please add a relevant basic test to test_dali_cpu_only.py for CPU only mode.

Reply:
done